pkg/stdlib/html/html_parse.go

```diff
@@ -0,0 +1,59 @@
+package html
+
+import (
+	"context"
+	"io/ioutil"
+
+	"github.com/MontFerret/ferret/pkg/drivers"
+	"github.com/MontFerret/ferret/pkg/runtime/core"
+	"github.com/MontFerret/ferret/pkg/runtime/values"
+	"github.com/MontFerret/ferret/pkg/runtime/values/types"
+)
+
+type HtmlParseParams struct {
+	FileName string
+	Driver   string
+}
+
+// HTML_PARSE loads an HTML page from a local file
+// By default, loads a page by http call - resulted page does not support any interactions.
+// @param params (Object) - Optional, An object containing the following properties :
+// 		driver (String) - Optional, driver name.
+// @returns (HTMLPage) - Returns loaded HTML page.
+func HTMLParse(ctx context.Context, args ...core.Value) (core.Value, error) {
+	err := core.ValidateArgs(args, 1, 2)
+
+	if err != nil {
+		return values.None, err
+	}
+
+	err = core.ValidateType(args[0], types.String)
+
+	if err != nil {
+		return values.None, err
+	}
+
+	var params PageLoadParams
+
+	if len(args) > 1 {
+		p, err := newPageLoadParams("about:blank", args[1])
+
+		if err != nil {
+			return values.None, err
+		}
+
+		params = p
+	}
+
+	b, err := ioutil.ReadFile(args[0].String())
```

Comment:
I would prefer to pass in content. I added an issue for IO function for such cases.

Reply:
Should I change it, or leave it as it is until the IO functions get implemented?

Reply:
I would change it. Pass content as a binary value.

When #395 gets closed - you will be able to read a file from disk and pass output to the function.

Reply:
@ziflex @patrikhl you can [read](https://github.com/MontFerret/ferret/pull/403) from disk disk now